src/kraft_controller.erl

```diff
@@ -1,16 +1,72 @@
 -module(kraft_controller).
 
+-moduledoc """
+Template-based controller handler for Kraft framework.
+
+This module provides a simplified controller for rendering HTML templates.
```

Comment:
I don't get this sentence. A controllers purpose is not to just "render HTML templates". It is to process the whole request/response cycle.